src/transformers/integrations.py

```diff
@@ -288,6 +298,49 @@ def dynamic_modules_import_trainable(*args, **kwargs):
     return best_run
 
 
+def run_hp_search_sigopt(trainer, n_trials: int, direction: str, **kwargs) -> BestRun:
+
+    from sigopt import Connection
+
+    conn = Connection()
+    proxies = kwargs.pop("proxies", None)
+    if proxies is not None:
+        conn.set_proxies(proxies)
+
+    experiment = conn.experiments().create(
+        name="huggingface-tune",
+        parameters=trainer.hp_space(None),
+        metrics=[
+            dict(name="objective", objective=direction, strategy="optimize"),
+        ],
+        parallel_bandwidth=1,
+        observation_budget=n_trials,
+        project="huggingface",
+    )
+    logger.info("created experiment: https://app.sigopt.com/experiment/" + experiment.id)
+
+    while experiment.progress.observation_count < experiment.observation_budget:
+        suggestion = conn.experiments(experiment.id).suggestions().create()
+        trainer.objective = None
+        trainer.train(resume_from_checkpoint=None, trial=suggestion)
+        # If there hasn't been any evaluation during the training loop.
+        if getattr(trainer, "objective", None) is None:
+            metrics = trainer.evaluate()
+            trainer.objective = trainer.compute_objective(metrics)
+
+        values = [
+            dict(name="objective", value=trainer.objective),
+        ]
+        obs = conn.experiments(experiment.id).observations().create(suggestion=suggestion.id, values=values)
+        logger.info("[suggestion_id, observation_id]: [%s, %s]" % (suggestion.id, obs.id))
```

Comment:
```suggestion
        logger.info(f"[suggestion_id, observation_id]: [{suggestion.id}, {obs.id}]")
```
We only use f-strings in the code base.

Reply:
fixed.